src/mame/atari/shuuz.cpp

```diff
@@ -184,14 +184,17 @@ uint32_t shuuz_state::screen_update(screen_device &screen, bitmap_ind16 &bitmap,
 					int const o13 = ((pf[x] & 0xf0) == 0xf0);
 
 					// compute the MO/PF signal
-					int mopf = 0;
-					if ((!(pf[x] & 0x80) && ((mo[x] & 0xc0) != 0xc0) && ((mo[x] & 0x0e) != 0x00) && !o13) ||
-						((pf[x] & 0x80) && ((mo[x] & 0xc0) == 0xc0) && ((mo[x] & 0x0e) != 0x00) && !o13))
-						mopf = 1;
+					int mopf = ((!(pf[x] & 0x80) && ((mo[x] & 0xc0) != 0xc0) && !o13) ||
+								 ((pf[x] & 0x80) && ((mo[x] & 0xc0) == 0xc0) && !o13));
```

Comment:
The MO/PF signal here no longer matches the equations from the GAL shown above.  Can you please link to the schematics for how this works?

Reply:
Sorry, I don’t have schematics evidence. My change is based on 
1) photo and video evidence of the game running which shows the shadow effect
2) the game code doing the work (preparing the high palette) to support the shadows
3) the game MO roms contain the sprite data to use the effect
4) the game looks correct when the effect is enabled
Here’s an additional reference video
https://www.instagram.com/p/CsWoVKPMyw8/